genesis_ledgers/berkeley.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
i don't think this is the genesis ledger we should be using, shouldn't it be test-snapps.json?  

rebase this branch off of `test-snapps-ledger` to get that ledger